examples/helm-inflator/local-configs/fn-config.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   annotations:
     config.kubernetes.io/function: |
       container:
-        image: gcr.io/kpt-functions/helm-inflator
+        image: gcr.io/kpt-functions/helm-template
```

Comment:
Can you manually push the helm-inflator image and avoid this change?

Reply:
Do I have the permission to directly push it?

Reply:
@frankfarzan Can you give Donny permissions for this?

Reply:
There should be different flows:

1- CI workflow should be testing the unreleased images not the latest.
2- CD should be testing the release candidate images

Why do you need to manually release an image? It seems like an antipattern, images should be released through automated CD.